### DISCUSSION TITLE
Draw Assist From Procreate

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#656] [Ideas]
I know this feature isn't planned for the big 2.0 update, whenever it's coming out, but it'd be convenient for us to be able to draw straight lines automatically, and have them perfectly placed, like in Procreate, so we done have to create square or rectangle shapes to get straight lines for flat surfaces for walk cycles. Also, if drawing grids were added, especially perspective grids, drawing our backgrounds in ToonSquid would be much simpler.

**Comment by inTIMidation297** [comment]
That already exists. https://youtu.be/EJi9BJ18eIU?si=3ymMDcZwBy4nKTb7

**Reply by DWrightAnimation** [reply]
That's not what I meant, I mean a feature where you turn on a drawing guide, then you turn on a draw assist mode, where you can only draw straight lines, without worrying about whether it's even or not.

**Reply by inTIMidation297** [reply]
Ohhh oops. Like symmetry tool and stuff. Yeah I would like that too

**Reply by otaworks** [reply]
It would be cool if the rulers were like this in Medibang/FireAlpaca:

https://www.youtube.com/watch?v=BGowZmyudM8&pp=ygUTbWVkaWJhbmcgcnVsZXIgdG9vbA%3D%3D

**Reply by otaworks** [reply]
This video may be a better explanation: 
https://www.youtube.com/watch?v=rfM8Gz9-QbM&pp=ygUVZmlyZWFscGFjYSBydWxlciB0b29s
